colossalai/engine/ophooks/_memtracer_ophook.py

```diff
@@ -105,7 +107,7 @@ class MemTracerOpHook(BaseOpHook):
         _count (int): the number of times the data file was written
         _data_prefix (string): the prefix of the stats data file
         _rank (int): the rank of current node
-    '''
+    """
```

Comment:
Can you change this to sphinx style?

Reply:
Ok, I will change this doc string to sphinx style.